pkg/apis/pingcap/v1alpha1/pd_config.go

```diff
@@ -33,26 +33,14 @@ import (
 // PDConfig is the configuration of pd-server
 // +k8s:openapi-gen=true
 type PDConfig struct {
-	// +optional
-	Version *bool `json:"-"`
-	// +optional
-	ConfigCheck *bool `json:"-"`
-
-	// +optional
-	ClientUrls string `toml:"client-urls,omitempty" json:"client-urls,omitempty"`
-	// +optional
-	PeerUrls string `toml:"peer-urls,omitempty" json:"peer-urls,omitempty"`
-	// +optional
-	AdvertiseClientUrls string `toml:"advertise-client-urls,omitempty" json:"advertise-client-urls,omitempty"`
-	// +optional
-	AdvertisePeerUrls string `toml:"advertise-peer-urls,omitempty" json:"advertise-peer-urls,omitempty"`
 
 	// +optional
 	Name string `toml:"name,omitempty" json:"name,omitempty"`
 	// +optional
 	DataDir string `toml:"data-dir,omitempty" json:"data-dir,omitempty"`
 	// +optional
 	ForceNewCluster *bool `json:"force-new-cluster,omitempty"`
+	// Optional: Defaults to true
```

Comment:
It seems that configuration items like `ForceNewCluster`, `EnableGRPCGateway ` are hidden.